lib/conjur/command.rb

```diff
@@ -100,6 +110,30 @@ def command_impl_for_list(global_options, options, args)
         end
       end
       
+      def validate_privileges message, &block
+        valid = begin
+          yield
+        rescue RestClient::Exception
```

Comment:
Should this check the status on the response from the API, too? If I make a typo in my resource name, I now get "You can't administer this record", rather than "404 Resource Not Found" (which seems more helpful).